other.html

```diff
@@ -164,7 +164,7 @@
        (28-Apr-2015)</LI>
 
   <LI> <A HREF="downloads/ihp2014mm.pdf">Overview of Metamath</A>
-         presentation by Norm Megill at Institut Henri Poincar� (2014)</LI>
+         presentation by Norm Megill at Institut Henri Poincar� (2014)</LI>
```

Comment:
There are a few occurrences where accents are replaced with "�". Is this intentional?

Reply:
Apparently the original was in Windows-1252, I've re-saved it as UTF-8. (Alternatively / additionally I could just use HTML escapes instead here.)

Reply:
I also checked for other non-ASCII, the only other file is mm_100.html and it is already UTF-8.

Reply:
Don't you need to change the line which says

```
<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=iso-8859-1">
```

if you are going to go with UTF-8? I do think UTF-8 is the best fix, we just might have to make sure the servers and/or browsers are treating things as UTF-8.